src/app/Icons/NavigationButtonInputIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function NavigationButtonBackIcon(): JSX.Element {
+  return (
+    <>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="-7 -6 35 35"
```

Comment:
Working with SVG was such a pain for me 🙈  I can see, that you used negative values in the viewbox and "90%" in the height/width. Would it also work, if you do the viewBox as 0 0 35 35 and give height/width a 100%?

Reply:
nooo, somehow not :) one is the position and the other one the size (or smth like that) and it will not stay in place if i cange the minus numbers :)